toasty/image.py

```diff
@@ -366,17 +372,30 @@ def load_path(self, path):
         # filetypes instead of just looking at extensions. But, lazy.
 
         if path.endswith('.npy'):
-            if self.desired_mode is not None:
-                mode = self.desired_mode
+            arr = np.load(path)
+            return Image.from_array(arr, default_format='npy')
+
+        if path.lower().endswith(('.fits', '.fts', '.fits.gz', '.fts.gz')):
+
+            # TODO: implement a better way to recognize FITS files
+            # TODO: decide how to handle multiple HDUs
```

Comment:
FWIW, I was envisioning that the ImageLoader class would have an "hdu_number" parameter for selecting the HDU of interest if needed. It would be mapped into a reusable `--hdu-number` argument for the command-line tools.

Reply:
I think simply ``hdu`` or ``hdu_ext`` would be better since it then allows HDU names to be passed, not just numbers?

Reply:
Sure!